### DISCUSSION TITLE
Re-use content of a section of another qmd-file with include

### DISCUSSION THREAD

**Discussion by friedeh** [quarto-dev/quarto-cli#971] [Q&A]
Hello, 

(and thank you for your great work on Quarto!)

I would like to re-use content of a section of another qmd-file with `{{< include _content.qmd >}}`, but not the whole text of the file.

Is there a way to do it? Is there a way to run something like the code below?

`{{< include _content.qmd#section >}`
or
`{{< include _content.qmd#sec-introduction >}`

Thank you
Hendrik

**Comment by jeremiahpslewis** [comment]
This would be a great feature to have for multi-language documentation, eg if you want to have Julia and R code in a single document and can’t because of 1 page == 1 kernel limitation (I know there are workarounds, but not really optimal ones).

**Comment by jjallaire** [comment]
No there isn't currently a way to do this but you make a good point about it being a great potential improvement!
